packages/api-explorer/src/JsonForm/arrays-editor.js

```diff
@@ -11,6 +11,7 @@ module.exports = (classReference) => class arrayCustomEditor extends classRefere
       if (
         this.jsoneditor &&
         this.jsoneditor.schema &&
+        this.jsoneditor.schema.required &&
         this.jsoneditor.schema.required.includes(this.key) &&
         Array.isArray(this.jsoneditor.schema.required)
```

Comment:
```suggestion
        Array.isArray(this.jsoneditor.schema.required) &&
        this.jsoneditor.schema.required.includes(this.key) &&
```
In this way could be better?

Reply:
better